frameworks/kafka/tests/test_jmx.py

```diff
@@ -0,0 +1,204 @@
+import pytest
+import random
+import string
+
+import sdk_cmd
+import sdk_tasks
+import sdk_install
+import sdk_security
+import sdk_utils
+import subprocess
+from tests import config
+
+
+def install_jmx_configured_kafka(self_signed_trust_store: bool = True):
+    foldered_name = sdk_utils.get_foldered_name(config.SERVICE_NAME)
+    sdk_install.uninstall(config.PACKAGE_NAME, foldered_name)
+    test_run = random_string()
+    create_keystore_cmd = ["keytool", "-genkey", "-alias", "self-signed-cert", "-keyalg",
+                           "rsa", "-dname", "CN=myhost.example.com,O=Example Company,C=US",
+                           "-keystore", "/tmp/{}-self-signed-keystore.ks".format(test_run), "-storepass", "deleteme",
+                           "-keypass", "deleteme", "-storetype", "jks"]
+
+    out = subprocess.check_output(create_keystore_cmd)
+    print(out)
+
+    create_keystore_cmd = ["keytool", "-list", "-v", "-keystore", "/tmp/{}-self-signed-keystore.ks".format(test_run),
+                           "-storepass", "deleteme"]
+
+    out = subprocess.check_output(create_keystore_cmd)
+    print(out)
+
+    write_to_file("deleteme", "/tmp/{}-keystorepass".format(test_run))
+    write_to_file("admin adminpassword", "/tmp/{}-passwordfile".format(test_run))
+    write_to_file("admin readwrite", "/tmp/{}-access".format(test_run))
+
+    sdk_security.install_enterprise_cli(False)
+
+    sdk_cmd.run_cli(
+        "security secrets create -f /tmp/{}-self-signed-keystore.ks /test/integration/kafka/keystore".format(
+            test_run))
+    sdk_cmd.run_cli(
+        "security secrets create -f /tmp/{}-passwordfile /test/integration/kafka/passwordfile".format(test_run))
+    sdk_cmd.run_cli(
+        "security secrets create -f /tmp/{}-keystorepass /test/integration/kafka/keypass".format(test_run))
+    sdk_cmd.run_cli("security secrets create -f /tmp/{}-access /test/integration/kafka/access".format(test_run))
```

Comment:
I wanted to suggest extracting this block of secrets code to its own method (like you did with `uninstall_jmx_secrets`) but I could see the argument that all this logic is self-contained including all the file paths.

Reply:
makes sense